gui/index.go

```diff
@@ -44,27 +60,42 @@ func (ui *GUI) GetIndex(w http.ResponseWriter, r *http.Request) {
 
 	data.Address = address
 
-	minedWork, err := ui.hub.FetchMinedWork(true)
+	if len(address) != 35 {
```

Comment:
Sorry, missed this earlier. Please decode the address and assert it's derived from the active network set in the config. Refer to: https://github.com/decred/dcrpool/blob/master/config.go#L443-L457

Reply:
added

Reply:
added